api/src/app.ts

```diff
@@ -20,6 +21,7 @@ const createApp = (clients: Clients, config: Config) => {
 
   app.use(morgan('short', { stream: logStream('http') }));
 
+  app.get('/all', addressablesController(clients, config));
```

Comment:
Doooo we want to keep this as `/addressables` or not? 
I don't have a strong opinion on this.

Reply:
It was `/all` in the RFC? I think that would be my preference because it describes better what the API is returning rather than what it has indexed (which doesn't necessarily make much sense to a consumer of the API without knowing its inner workings). I think I've just largely reverse engineered that opinion though

Reply:
(and `all` in the pipeline means reindex all indices, but you're right that we can do the distinction at that point)